db/migrate/20230206182727_change_template_to_text.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ChangeTemplateToText < ActiveRecord::Migration[7.0]
+  def change
+    change_column :form_templates, :template, :text
```

Comment:
Since this isn't merged yet, it would be cleaner to skip this second migration, and add the column type in `db/migrate/20230202173641_create_form_templates.rb`. Would need some resetting in your local environment.